tools/isobuild/import-scanner.ts

```diff
@@ -812,16 +837,16 @@ export default class ImportScanner {
     });
   }
 
-  _getSourcePath(file) {
+  private getSourcePath(file: File) {
     let sourcePath = file.sourcePath;
     if (sourcePath) {
       if (pathIsAbsolute(sourcePath)) {
+        let relPath: string | undefined;
         try {
-          var relPath = pathRelative(this.sourceRoot, sourcePath);
-
+          relPath = pathRelative(this.sourceRoot, sourcePath);
         } finally {
           if (! relPath || relPath.startsWith("..")) {
-            if (this.resolver._joinAndStat(this.sourceRoot, sourcePath)) {
```

Comment:
This would have been a bug, since we renamed `_joinAndStat` to `joinAndStat` and made it private in 4334fd4ceb447506f0a0802c1a0e05f529037e8e. Thanks TypeScript!